server/repository/src/migrations/v2_06_00/add_name_next_of_kin_id.rs

```diff
@@ -16,7 +16,7 @@ impl MigrationFragment for Migrate {
             "#
         )?;
 
-        // Reset translate all items on the next sync
+        // Reset translate all names on the next sync
```

Comment:
Not resetting integration_datetime in the new migration seeing as it is already in this one 😁